src/main/resources/application-dev.yaml

```diff
@@ -139,6 +139,15 @@ no.nav.security.jwt:
           client-id: ${AZURE_APP_CLIENT_ID}
           client-secret: ${AZURE_APP_CLIENT_SECRET}
           client-auth-method: client_secret_basic
+      modia-context-holder-onbehalf-of:
```

Comment:
Har glemt config for application-e2e.yaml, som jeg tror brukes av e2e løypa i tilbakekreving

Reply:
Nei. Det skal gå greit